src/main/java/datastructs/CategoricalSample.java

```diff
@@ -29,13 +32,13 @@ public CategoricalSample(String name, List<String> data){
 	 /**
      * The name of the sample
      */
-    public final String getName(){return name_;}
```

Comment:
why did you opt out `getName`? seems more intuitive here. does the underscore after name means sth?

Reply:
We I don't see the reason to have this ```get...``` that's all :) :)

Reply:
perfect!